src/freenet/pluginmanager/PluginManager.java

```diff
@@ -820,66 +806,16 @@ public void unregisterPluginToadlet(PluginInfoWrapper pi) {
 		}
 	}
 
-	public void addToadletSymlinks(PluginInfoWrapper pi) {
```

Comment:
I did a search with grep in my work directory and even though I don’t claim to have all plugins’ source code I got no hits in anywhere but the plugin manager. I will deprecate them but I would like to schedule date for removal.